### PR TITLE
[kjobctl] Fix output Slurm variables

### PR DESCRIPTION
**Pull request by IrvingMg**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:
Fix issue with output Slurm variables that don't take the calculated value.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/experimental/kjobctl/pkg/cmd/create/create_test.go [hunk 1]
```diff
@@ -699,7 +699,7 @@ func TestCreateCmd(t *testing.T) {
 			beforeTest: beforeSlurmTest,
 			afterTest:  afterSlurmTest,
 			args: func(tc *createCmdTestCase) []string {
-				return []string{"slurm", "--profile", "profile", "--", tc.tempFile}
+				return []string{"slurm", "--profile", "profile", "--", "--cpus-per-task", "2", tc.tempFile}
```
Comment (mbobrovskyi) [inline cmd/experimental/kjobctl/pkg/cmd/create/create_test.go:702]: Is this flag required? This test is more useful for Slurm with minimal flags.

Comment (IrvingMg) [reply cmd/experimental/kjobctl/pkg/cmd/create/create_test.go:702]: I added it in there to take advantage of this test case and avoid duplicating the full init-script but I can add a new test specifically for this flag.

Comment (IrvingMg) [reply cmd/experimental/kjobctl/pkg/cmd/create/create_test.go]: Moved flag to a different test case and fix calculations when n > 1.

**Review by trasc — COMMENTED** [review]
/approve

**Pull request merged**